astropy/visualization/wcsaxes/core.py

```diff
@@ -194,9 +197,9 @@ def imshow(self, X, *args, **kwargs):
         else:
             if isinstance(X, Image) or hasattr(X, 'getpixel'):
                 X = X.transpose(FLIP_TOP_BOTTOM)
-                kwargs['origin'] = 'lower'
+                origin = 'lower'
```

Comment:
Is this line necessary?  It seems like `origin='lower'` always at this point (assuming the valid options are `'lower, 'upper', or None`).